content/Arduino Cloud/General Cloud/Wi-Fi-troubleshooting.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Wi-Fi troubleshooting"
+---
+
+This guide is intended to help you troubleshoot common Wi-Fi connectivity issues with your devices. Follow the steps below to identify and resolve potential issues.
+
+## 1. Frequency Bands: 2.4 GHz vs 5 GHz
+
+### Use the 2.4 GHz Band for IoT Devices
+
+Arduino boards and most IoT devices work better on the 2.4 GHz band. The 2.4 GHz band offers better range and signal strength through obstacles compared to 5 GHz.
```

Comment:
```suggestion
Arduino boards and most IoT devices can only be used with 2.4 GHz band networks. The 2.4 GHz band offers better range and signal strength through obstacles compared to 5 GHz.
```

The previous "work better" statement implies that they also work with 5 GHz networks, only not as well. That is false. It is essential to clearly communicate that the 2.4 GHz band _must_ be used.